mkosi/resources/mkosi.md

```diff
@@ -980,6 +980,11 @@ boolean argument: either `1`, `yes`, or `true` to enable, or `0`, `no`,
     images as self-signed binaries would not be accepted by the signed
     version of shim.
 
+`BootloaderEntryFormat=`, `--bootloader-entry-format`
```

Comment:
This needs to document the default and that the default is special, i.e. that it automatically picks up the count and hash.